mscore/inspector/inspectorBase.cpp

```diff
@@ -544,7 +544,11 @@ void InspectorBase::mapSignals(const std::vector<InspectorItem>& il, const std::
                         resetButton->setIcon(*icons[int(Icons::reset_ICON)]);
                         connect(resetButton, &QToolButton::clicked, [=] { resetClicked(i); });
                         Sid sidx = inspector->element()->getPropertyStyle(ii.t);
-                        if (sidx != Sid::NOSTYLE) {
+                        // S button for fingering placement is bugged and proposed to be hidden
+                        // it can be brought back once the values of fingering placement are changed to
+                        // "Auto, Above, Below", "Auto" as default
+                        // See https://musescore.org/en/node/288372 and https://musescore.org/en/node/297092
+                        if (sidx != Sid::NOSTYLE && sidx != Sid::fingeringPlacement) {
```

Comment:
No doubt this works, I wonder if there is something less hacky that could be used here?  Like maybe some flag we set in the fingering inspector itself (meaning, of course, we reinstate it)

Reply:
I don't see a way to do that since the S button is almost only related to `sidx`.

Reply:
What if there were an additional field, "allowSetAsStyle", defaults to true, within InspectorItem?  Then the fingering inspector could explicitly set it to false, so could any other settings that eventually need this.

Reply:
Good idea.

Reply:
Well, as long as `InspectorFingering` directly inherits `InspectorStaffText`, this is not possible, because the `iiList` (the list of `InspectorItem`) is defined within the construction of the inspector class.

Reply:
Right, that’s why I said “meaning, of course, we reinstate it”.  Although, doing the special case in InspectorStaffText (setting that property based on the actual element type) wouldn’t bother me.

Reply:
Doee "reinstate" mean making it no longer related to `InspectorStaffText`?

Reply:
Well, either that or maybe make it a subclass.  Or, as I also suggested, you could just do the test in InspectorStaffText.

Reply:
Done with the last!